plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/AbstractTestIcebergConnectorTest.java

```diff
@@ -123,6 +123,7 @@ protected boolean hasBehavior(TestingConnectorBehavior connectorBehavior)
                 return false;
 
             case SUPPORTS_CREATE_MATERIALIZED_VIEW:
+            case SUPPORTS_CREATE_VIEW:
```

Comment:
keep ordering as in `TestingConnectorBehavior` (view before mv), and maybe keep cases separate as Views and MVs are separate features (although related)